concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limit/Measurement.java

```diff
@@ -11,7 +11,7 @@
      * @param sample
      * @return True if internal state was updated
```

Comment:
comment wrong now

Reply:
Also seems to change the outcome a bit, it's not longer possible to tell if the add actually happened? Does that matter?

Reply:
No, it no longer matters.